deps/build.jl

```diff
@@ -21,4 +21,10 @@ end
     provides(Homebrew.HB, "hdf5", hdf5, os = :Darwin )
 end
 
+begin
+    hdf5 = library_dependency("libhdf5")
```

Comment:
right, but judging by the appveyor results, it looks like creating two separate instances of it is breaking winrpm